plugin/trino-kudu/src/main/java/io/trino/plugin/kudu/properties/KuduTableProperties.java

```diff
@@ -158,6 +158,11 @@ else if (!hashColumns2.isEmpty()) {
             design.setRange(new RangePartitionDefinition(rangeColumns));
         }
 
+        // un-partitioned table
+        if (!design.hasPartitions()) {
+            design.setRange(new RangePartitionDefinition(ImmutableList.of()));
```

Comment:
We could extract `new RangePartitionDefinition(ImmutableList.of())` as `EMPTY_RANGE_PARTITION` constant.